client/src/client.rs

```diff
@@ -442,6 +444,15 @@ impl<R: Reactor> Handle<R> {
 
         Ok(())
     }
+
+    /// Get number of connected peers.
+    pub fn get_connected_peers(&self) -> Result<HashSet<SocketAddr>, handle::Error> {
+        let (sender, recvr) = chan::bounded(1);
+        self.command(Command::GetPeers(sender))?;
+        let res = recvr.recv()?;
+
+        Ok(res)
+    }
```

Comment:
A few comments on this:

* Comment should be updated
* I think we should call this `get_peers` to match the name of the command
* We can combine the last two lines (see `get_tip`)

Reply:
Fixed in 5ecdbf6